smart-contracts/contracts/mixins/MixinLockCore.sol

```diff
@@ -165,7 +165,7 @@ contract MixinLockCore is
     address _beneficiary
   ) external
   {
-    require(msg.sender == beneficiary, 'ONLY_BENEFICIARY');
+    require(msg.sender == beneficiary || isLockManager(msg.sender), 'ONLY_BENEFICIARY_OR_LOCKMANAGER');
```

Comment:
Hmm should we maybe update the onlyOwnerOrBeneficiary modifier instead so that withdrawal gets the fix too?

Reply:
It makes sense, but I wonder if it's worth it to even have a modifier for this if it's only used twice?

Reply:
heh yes! I question modifier best practices as well. It might be nice to do a quick test to understand the tradeoffs - but we can revisit that.

Reply:
sounds good